fdbserver/VersionedBTree.actor.cpp

```diff
@@ -5687,6 +5825,12 @@ class VersionedBTree {
 			}
 			return true;
 		}
+
+		void finish(IPageEncryptionKeyProvider* keyProvider, bool useEncryptionDomain) {
```

Comment:
Here again these args do not change from call to call so I think members are more appropriate for their purpose.

Reply:
Also, previously the encoding type was not stored, I was passing it to the constructor and therefore it is also needed in `next()` but since `splitPages()` will always pass the same value it may be cleaner to make it a member as well.